config/routes.rb

```diff
@@ -15,7 +15,7 @@
   end
 
   get "/logout", to: "sessions#destroy", as: :destroy_admin_user_session
-  resource :session, only: %i[new create destroy]
+  resource :sessions, only: %i[new create destroy]
```

Comment:
Resources should be plural

Reply:
Explained here https://github.com/CruGlobal/ararat/pull/42#discussion_r470376478 why I'd like to keep it singular

Reply:
In that case, should line 17 be singular as well?

Reply:
Nope, guide says "singular resources map to plural controllers"